serpapi/serp_api_client.py

```diff
@@ -61,11 +61,11 @@ def get_results(self, path='/search'):
         """
         return self.get_response(path).text
 
-    def get_html(self):
+    def get_html(self, path='/search.html'):
         """Returns:
             Raw HTML search result from Google
         """
-        return self.get_results()
+        return self.get_response(path).text
```

Comment:
Let's use the same approach as in the `get_json` method.

```suggestion
    def get_html(self):
        """Returns:
            Raw HTML search result from Google
        """
        self.params_dict["output"] = "html"
        return self.get_results()
```